plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways.tsx

```diff
@@ -127,7 +128,16 @@ export const PaymentGateways = () => {
 				) }
 			</>
 		),
-		after: <Gridicon icon="chevron-right" />,
+		after: (
+			<a
+				href={
+					ADMIN_URL +
+					'admin.php?page=wc-settings&tab=checkout&section=offline'
```

Comment:
nit: Can we use `getAdminLink` here?

Reply:
Yes, thanks for the suggestion! Done in 814e3c2